cmd/contour/contour.go

```diff
@@ -181,7 +187,35 @@ func main() {
 		et := &contour.EndpointsTranslator{
 			FieldLogger: log.WithField("context", "endpointstranslator"),
 		}
-		k8s.WatchEndpoints(&g, client, wl, et)
+		coreInformers.Core().V1().Endpoints().Informer().AddEventHandler(et)
+
+		g.Add(func(stop <-chan struct{}) error {
+			log := log.WithField("context", "coreinformers")
+
+			log.Println("waiting for cache sync from core informers")
+			coreInformers.WaitForCacheSync(stop)
+
+			log.Println("starting core shared informer")
```

Comment:
log.Println("started") is probably good enough given the context captured in the logger.